docs/tutoriales/ejecutar-boilerplate.md

```diff
@@ -0,0 +1,48 @@
+---
+id: ejecutar-boilerplate
+title: Ejecutar Boilerlate
+sidebar_label: Ejecutar Boilerplate
+---
+
+# Ejecutando el Boilerplate
+
+## Herramientas
+
+Para la ejecución del Boilerplate se requieren las siguientes herramientas:
+
+- [git](https://git-scm.com/)
+- [node.js](https://nodejs.org/es/)
+- [docker](https://www.docker.com/)
+- [docker-compose](https://docs.docker.com/compose/)
```

Comment:
I think in planning meeting, comment on adding how run in Windows and Linux section

Reply:
In this case of doing the installation in ubuntu, it is not necessary to install Hapi, Postgres, and Hasura as git, node, and docker because docker images are being used. But it's needed for windows and probably mac. Yes, this must be clarified by doing different sections.